syntaxes/ocaml.json

```
@@ -126,12 +126,12 @@
           "match": "\\(\\*\\*\\)"
         },
         {
-          "comment": "ocamldoc comment",
+          "comment": "odoc comment",
           "name": "comment.doc.ocaml",
           "begin": "\\(\\*\\*",
           "end": "\\*\\)",
           "patterns": [
-            { "include": "source.ocaml.ocamldoc#markup" },
+            { "include": "source.ocaml.odoc#markup" },
```

If you apply the change from https://github.com/ocamllabs/vscode-ocaml-platform/pull/756/files#r756447787, you should make this just

```suggestion
            { "include": "source.ocaml.odoc" },
```

This way the documentation tags will also be highlighted in OCaml odoc comments.